modules/deploy/pages/redpanda/manual/resilience/shadowing-guide.adoc

```diff
@@ -0,0 +1,208 @@
+
+[[verify-functionality]]
+=== Verify application functionality
+
+Test critical application workflows:
+
+[,bash]
+----
+# Verify applications can produce messages
+rpk topic produce <critical-topic> --brokers <shadow-cluster-brokers>
+
+# Verify applications can consume messages
+rpk topic consume <critical-topic> --brokers <shadow-cluster-brokers> --num 1
+----
+
+Test message production and consumption, consumer group functionality, and critical business workflows to ensure everything is working properly.
+
+[[cleanup-stabilize]]
+=== Clean up and stabilize
+
+Once applications are running normally:
+
+[,bash]
+----
+# Optional: Delete the shadow link (no longer needed)
+rpk shadow delete <disaster-recovery-link-name> --no-confirm
+----
+
+Document the time of failover initiation and completion, applications affected and recovery times, data loss estimates based on replication lag, and issues encountered during failover.
+
+== Troubleshooting common issues
+
+=== Shadow link in PAUSED state
+
+**Problem**: Shadow link shows `PAUSED` instead of `ACTIVE`
+
+**Solution**:
+[,bash]
+----
+# Resume the shadow link
+rpk shadow resume <disaster-recovery-link-name>
+
+# Wait for it to return to ACTIVE state
+rpk shadow status <disaster-recovery-link-name>
+----
+
+=== Topics stuck in FAILING_OVER state
+
+**Problem**: Topics remain in `FAILING_OVER` state for extended periods
+
+**Solution**: Check shadow cluster logs for errors and ensure sufficient resources (CPU, memory, disk) are available on the shadow cluster. Verify network connectivity between shadow cluster nodes.
+
+If topics remain stuck and you need to fail over everything immediately, you can force delete the shadow link, which will promote all topics:
+
+[,bash]
+----
+# Force delete the shadow link to promote all topics
+rpk shadow delete <disaster-recovery-link-name> --force
+----
+
+[WARNING]
+====
+Force deleting a shadow link immediately promotes all topics in the link. This action is irreversible and should only be used when topics are stuck and you need immediate access to all replicated data.
+====
+
+=== Application connection failures
+
+**Problem**: Applications cannot connect to shadow cluster after failover
+
+**Solution**: Verify shadow cluster broker endpoints are correct and check security group and firewall rules. Confirm authentication credentials are valid for shadow cluster and test network connectivity from application hosts.
+
+=== Consumer group offset issues
+
+**Problem**: Consumers start from beginning or wrong positions
+
+**Solution**: Verify consumer group offsets were replicated (check your filters) and use `rpk group describe <group-name>` to check offset positions. If necessary, manually reset offsets to appropriate positions.
```

Comment:
Can you add a link to the doc they can follow for manually resetting consumer group offsets?

https://support.redpanda.com/hc/en-us/articles/23499121317399-How-to-manage-consumer-group-offsets-in-Redpanda